src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ExportInterceptingPropertyValueProviderAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    /// <summary>
+    /// Exports a <see cref="IInterceptingPropertyValueProvider"/> extension to CPS.
+    /// </summary>
+    [MetadataAttribute]
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface, AllowMultiple = false, Inherited = false)]
+    public sealed class ExportInterceptingPropertyValueProviderAttribute : ExportAttribute
```

Comment:
The attribute is to keep this consistent with [ExportDynamicEnumValuesProviderAttribute](http://index/Microsoft.VisualStudio.ProjectSystem/R/9268c241811fc976.html), which has a similar functionality. We should potentially consider moving this intercepting value provider down to the CPS layer, as this seems like generic functionality.